cvat/apps/onepanelio/views.py

```diff
@@ -208,8 +208,10 @@ def dump_training_data(uid, db_task, stamp, dump_format, cloud_prefix, request):
             import boto3
             from botocore.exceptions import ClientError
 
-            #check if datasets folder exists on aws bucket
-            s3_client = boto3.client('s3')
+            if endpoint != 's3.amazonaws.com':
+                s3_client = boto3.client('s3', endpoint_url='https://'+endpoint)
```

Comment:
@savan77 scheme should only be prepended if there is no scheme in `endpoint`. For Minio, the user might put in the scheme.

Reply:
Ok, should I expect it to be `https` or `http` for Minio? I can make changes accordingly.